mathematics/bisection.r

```diff
@@ -0,0 +1,25 @@
+
+f <- function(x){
+  x^3-x-2
+}
+
```

Comment:
Since the main focus is the bisection algorithm, include that on the top and add any supplementary functions below it.